src/main/java/redis/clients/jedis/commands/MultiKeyBinaryRedisPipeline.java

```diff
@@ -30,15 +29,19 @@
 
   Response<byte[]> lmove(byte[] srcKey, byte[] dstKey, ListDirection from, ListDirection to);
 
-  Response<byte[]> blmove(byte[] srcKey, byte[] dstKey, ListDirection from, ListDirection to, int timeout);
+  Response<byte[]> blmove(byte[] srcKey, byte[] dstKey, ListDirection from, ListDirection to, double timeout);
 
   Response<List<byte[]>> blpop(byte[]... args);
 
+  Response<List<byte[]>> blpop(double timeout, byte[]... args);
+
   Response<List<byte[]>> brpop(byte[]... args);
 
-  Response<KeyedTuple> bzpopmax(int timeout, byte[]... keys);
+  Response<List<byte[]>> brpop(double timeout, byte[]... args);
```

Comment:
@sazzad16 I think you should deprecate the interface methods and not only the implementation methods

Reply:
@gkorland The deprecated methods were not part of any interface.